scripts/start-kind-local-registry.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
What does this file do?

Reply:
this is a script to run a local registry ( i.e. docker hub ): https://kind.sigs.k8s.io/docs/user/local-registry/
In this way I can easily develop and test in the development phase with no need to push my docker images to a remote registry